packages/kamino-sdk/src/Kamino.ts

```diff
@@ -282,6 +284,50 @@ export class Kamino {
     }
   }
 
+  /**
+   * Get amount of specified token in all Kamino live strategies
+   * @param tokenMint token mint pubkey
+   */
+  async getTotalTokensInStrategies(tokenMint: PublicKey | string): Promise<TotalStrategyVaultTokens> {
+    // TODO: update and use this method call to only fetch live strategies once strategy status on-chain data is ready
+    // const strategies = await this.getAllStrategiesWithFilters({
+    //   strategyType: undefined,
+    //   strategyCreationStatus: undefined,
+    // });
+    const strategies = await this.getStrategies(this._config.kamino.liveStrategies.map((x) => x.address));
+    let totalTokenAmount = new Decimal(0);
+    const vaults: StrategyVaultTokens[] = [];
+    for (const liveStrategy of this._config.kamino.liveStrategies) {
+      const strategy = strategies.find(
+        (x) =>
+          x?.sharesMint.toString() === liveStrategy.shareMint.toString() &&
+          (x.tokenAMint.toString() === tokenMint.toString() || x.tokenBMint.toString() === tokenMint.toString())
+      );
+      if (!strategy) {
+        continue;
+      }
+
+      const balances = await this.getStrategyBalances(strategy);
+      const aTotal = balances.computedHoldings.invested.a.plus(balances.computedHoldings.available.a);
```

Comment:
not sure if `.a` is w decimals or without decimals, but just saying to make sure we return human readable (and that they match the ui anyway)

Reply:
These two are already in human readable format with decimals which is what we want.